components/Link.jsx

```diff
@@ -1,22 +1,32 @@
 import React, { PropTypes } from 'react'
+import { Link as ReactRouterLink } from 'react-router'
```

Comment:
This means we, who don't use *react-router*, would bundle the _entire_ dependency until Webpack 2.0 with tree-shaking is official. Right?